test/unit/src/common/components/select-repository-list/t_select-repository-list.js

```diff
@@ -92,7 +92,7 @@ describe('<SelectRepositoryListComponent /> instance', function() {
     });
 
     it('should render disabled Add button', function() {
-      expect(wrapper.find('Button').prop('disabled')).toBe(true);
+      expect(wrapper.find('#add-repo').prop('disabled')).toBe(true);
```

Comment:
I would prefer to avoid adding `id`s to elements just for test purposes.

We could use enzyme API to find a button we need, for example `wrapper.find('Button').last().prop('disabled')` should work in this case (as add button is last one).
Of course it's not perfect (breaks if buttons will be moved around in the component) but lets us avoid unnecessary element ids.